drivers/i2c/slave/Kconfig

```diff
@@ -0,0 +1,65 @@
+# Kconfig - I2C Slave configuration options
+
+#
+# Copyright (c) 2017 BayLibre, SAS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# I2C options
+#
+menuconfig I2C_SLAVE
+	bool
+	prompt "I2C Slave Drivers"
```

Comment:
Either use the prompt or not. Don't mix both in the same Kconfig. 

Actually, my personal preference would go for just <type> "message to prompt" and not using prompt keyword.

Reply:
I copied it from i2c verbatim, i will fix this